Libraries/LibGfx/PainterSkia.cpp

```diff
@@ -142,6 +142,21 @@ void PainterSkia::fill_rect(Gfx::FloatRect const& rect, Color color)
     });
 }
 
+static SkMatrix to_skia_matrix(Gfx::AffineTransform const& affine_transform)
```

Comment:
You have this here and in `DisplayListPlayerSkia.cpp` - move it to `SkiaUtils` instead.